docs/sources/mimir/migration-guide/migrating-from-cortex.md

```diff
@@ -200,6 +201,9 @@ You can update to the Grafana Mimir Helm chart from the Cortex Helm chart.
       enabled: true
   ```
 
+  Ingester needs storage for the WAL and to form blocks for upload. The WAL was optional in Cortex with chunks, but not optional in Mimir.
+  A statefulset is the most convenient way to arrange that each pod gets a storage volume.
```

Comment:
I'd remove this sentence, we use statfulset to have consistent naming of pods and therefore consistent hashing . But it's all implementation detail the user does not have to know.

Reply:
Yes, I can add naming consistency is also other reason I forgot to add. Should I add?

I am thinking to add the persistence requitement because I was curious myself why we want to use statefulset. User doesn't need to know the internal detail indeed, but I believe this information still useful.

Do you still think I should remove this whole paragraph?

Reply:
agreed IRL that this can stay